test/core/promise/activity_test.cc

```diff
@@ -360,25 +360,28 @@ TEST(AtomicWakerTest, ThreadStress) {
   AtomicWaker waker;
 
   threads.reserve(90);
-  for (int i = 0; i < 30; i++) {
+  for (int i = 0; i < 5; i++) {
     threads.emplace_back([&] {
       while (!done.load(std::memory_order_relaxed)) {
         waker.Wakeup();
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
My main concerns is if this indicates we're going to need similar hacks in production code. The activity_test is timing out on Windows, so adding a sleep to every thread seems like maybe this test is now hacking around a race in production code? https://source.cloud.google.com/results/invocations/f3604bbf-18b8-47c9-b758-e425a9b7e62c/targets/%2F%2Ftest%2Fcore%2Fpromise:activity_test/log

Reply:
I think not... this shows up because we're in a tight loop around an atomic on a platform that doesn't tend to be the fairest scheduler.

We can't fix the fairness, but generally we're not in these tight loops.

Reply:
Fair enough.